### DISCUSSION TITLE
[ERROR] Download error: Unable to download episode

### DISCUSSION THREAD

**Discussion by greenyellowl** [anidl/multi-downloader-nx#759] [Q&A] [answered]
Hello
I downloaded and unpacked the build
I downloaded the dependencies and edited the bin-path.yml

ffmpeg: "C:/Users/Helen/Downloads/multi-downloader-nx-windows-gui/multi-downloader-nx-windows-x64-gui/ffmpeg.exe"
mkvmerge: "C:/Program Files/MKVToolNix/mkvmerge.exe"
ffprobe: "C:/Users/Helen/Downloads/multi-downloader-nx-windows-gui/multi-downloader-nx-windows-x64-gui/ffprobe.exe"
mp4decrypt: "C:/Users/Helen/Downloads/multi-downloader-nx-windows-gui/multi-downloader-nx-windows-x64-gui/mp4decrypt.exe"

Now I'm trying to download the series

[DEBUG] Got download options: {"id":"GRGGPG93R","q":0,"e":"1","dubLang":["eng"],"dlsubs":["en"],"fileName":"[${service}] ${showTitle} - S${season}E${episode} [${height}p]","dlVideoOnce":false,"all":false,"but":false,"noaudio":false,"novids":false,"simul":false,"ids":["GYQ4M4216"],"title":"Fullmetal Alchemist","parent":{"title":"Fullmetal Alchemist: Brotherhood","season":"1"},"image":"https://www.crunchyroll.com/imgsrv/display/thumbnail/1200x675/catalog/crunchyroll/025260e7ab620e093bff1233dd78629d.jpg","episode":"1"}

But I keep getting an error

[S1E1] - Fullmetal Alchemist [✓ English]
Requesting: [GYQ4M4216] Fullmetal Alchemist: Brotherhood - 1 - Fullmetal Alchemist
Chapter request successful
[WARN] Decryption not enabled!
[WARN] No full streams found!
[ERROR] Download error: Unable to download episode 1 from GRGGPG93R

What am I doing wrong?

**Comment by Jaynator495** [comment, marked as answer]
You need a CDM in the widevine directory

**Reply by RexGhidorah** [reply]
Hi!
Im getting the same error but I got a valid cdm. :,D
![image](https://github.com/user-attachments/assets/374aec6f-bd73-4240-b8c7-0e4a30b41202)
